### ISSUE TITLE
HTML file structure

### ISSUE THREAD

**Issue by kortenkamp** [CindyJS/CindyJS#331]
Currently, the files created by Cinderella as well as the examples in the repo have the script code (both CindyScript and JavaScript) in the `<head>` tag. Wouldn't it be more appropriate to have them in the `<body>`, as a continuous block, in particular to enable easier copy & paste into other web pages?

**Comment by gagern** [comment]
Makes sense, and I see no reason against this. Should we put the `<canvas>` element before the `<script>` elements? I think I'd prefer to have `<canvas>` first, then CindyScript and the JavaScript.

Should we allow omitting the `canvas` element altogether, and let the `createCindy` call write an element to the document in such situations? I'd say no, since I dislike scripts writing HTML, but as long as I'm not firced to use this, it might be a viable way to cut back a bit on the boilerplate code.

**Comment by kranich** [comment]
`canvas` has become `div` since 1d1e6d70e86a1c609795aff6e20a75f2db17948e
